src/pybind/mgr/orchestrator/module.py

```diff
@@ -850,10 +868,8 @@ def _apply_misc(self, service_type=None, placement=None, unmanaged=False, inbuf=
             content: Iterator = yaml.load_all(inbuf)
             specs = [ServiceSpec.from_json(s) for s in content]
         else:
-            placement = PlacementSpec.from_string(placement)
-            placement.validate()
-
-            specs = [ServiceSpec(service_type, placement=placement, unmanaged=unmanaged)]
+            placement = PlacementSpec.from_string(placement)  # type: ignore
+            specs = [ServiceSpec(service_type, placement=placement, unmanaged=unmanaged)]  # type: ignore
```

Comment:
again. you should not get an error here.

Reply:
https://github.com/ceph/ceph/pull/34843/commits/f132798b4c1b2f87af4205257daef020f9ab30fb